showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -10,6 +10,7 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 				[dateFilter]="customDateFilter"
 				[startDate]="startDate"
 				startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
+				[dateMask]="true"
```

Comment:
Can you add another example for custom date masks?

Reply:
I would suggest to divide the examples in 2 groups:

1.- Simple date range picker (normal config with no mask... so basically what we previously had)
2.- With date masks (one date-range-picker with default mask and another date-range-picker with custom mask)